plugin-energy-estimation-tdp/src/lib.rs

```diff
@@ -48,48 +48,37 @@ impl AlumetPlugin for EnergyEstimationTdpPlugin {
 
     fn init(config: ConfigTable) -> anyhow::Result<Box<Self>> {
         let config = deserialize_config(config)?;
-        Ok(Box::new(EnergyEstimationTdpPlugin {
-            config: Some(config),
-            metrics: Arc::new(Mutex::new(Metrics::default())),
-        }))
+        Ok(Box::new(EnergyEstimationTdpPlugin { config }))
     }
 
     fn start(&mut self, alumet: &mut alumet::plugin::AlumetPluginStart) -> anyhow::Result<()> {
-        let mut metrics = self.metrics.lock().unwrap();
-        let config: Config = self.config.take().unwrap();
-
         // Create the energy attribution metric and add its id to the
         // transform plugin metrics' list.
-        metrics.pod_estimate_attributed_energy = Some(alumet.create_metric(
+        let pod_estimate_attributed_energy_metric = alumet.create_metric(
             "pod_estimate_attributed_energy",
             Unit::Joule,
-            "Energy consumption estimated to the pod",
-        )?);
+            "Pod's estimated energy consumption",
+        )?;
 
-        // Add the transform now but fill its metrics later.
-        alumet.add_transform(Box::new(EnergyEstimationTdpTransform::new(
-            config,
-            self.metrics.clone(),
-        )));
-        Ok(())
-    }
+        let cpu_usage = self.config.as_ref().unwrap().cpu_usage_per_pod.clone();
+        let config_cpy = self.config.take().unwrap();
```

Comment:
```suggestion
        let config = self.config.take().unwrap();
```
It's not a copy anymore :)

Reply:
done